dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops

```diff
@@ -54,3 +54,35 @@ CWMOT5004.factory.used.without.metadata.useraction=Review your code to see why t
 CWMOT5005.mptelemetry.unknown.log.source=CWMOT5005W: The MicroProfile Telemetry logging feature does not recognize the [ {0} ] log source.
 CWMOT5005.mptelemetry.unknown.log.source.explanation=The Liberty runtime environment does not provide the specified log source.
 CWMOT5005.mptelemetry.unknown.log.source.useraction=Check the configuration information and specify one of the supported sources for MicroProfile Telemetry.
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Monitor Metrics messages
+#-----------------------------------------------------------------------------------------------------------------------------
+
+threadpool.activeThreads.description=The number of threads that are actively running tasks.
+threadpool.size.description=The size of the thread pool.
+
+session.activeSessions.description=The number of concurrently active sessions. A session is active when the runtime is processing a request that uses that user session.
+session.created.description=The number of sessions logged in since this metric was enabled.
+session.invalidated.description=The number of sessions logged out since this metric was enabled.
+session.invalidatedbyTimeout.description=The number of sessions logged out because of a timeout since this metric was enabled.
+session.live.description=The number of users that are currently logged in.
+
+connectionpool.handle.count.description=The number of connections that are in use. This number might include multiple connections that are shared from a single managed connection.
+connectionpool.connection.created.description=The total number of managed connections that were created since the pool creation.
+connectionpool.connection.destroyed.description=The total number of managed connections that were destroyed since the pool creation.
+connectionpool.connection.free.description=The number of managed connections that are available.
+connectionpool.connection.waitTime.description=The amount of time that connection requests waited for a connection.
```

Comment:
```suggestion
connectionpool.connection.waitTime.description=The total amount of time that all connection requests waited for a connection.
```

Reply:
This makes it sound like all connections had to wait at some point. Not all connections need to wait. 
Can we remove the `all` ?